test/compilation/compilation_test.c

```diff
@@ -4,9 +4,86 @@
 
 #include <fizzy/fizzy.h>
 
-bool dummy(void);
+bool validate(const uint8_t* binary, size_t size);
```

Comment:
For the future, the "compilation test" is to check if the code in the header is valid `C99`, `C11`. The is no need to add code to it.

Reply:
I know, but I can imagine the case when the header compiles, but then calling the function doesn't compile in C like you imagine it would.

Reply:
There are unittests for that.

Reply:
The ones compiled in `C99` and `C11`?